src/Exceptionless.AppHost/Program.cs

```diff
@@ -0,0 +1,55 @@
+var builder = DistributedApplication.CreateBuilder(args);
+
+var elastic = builder.AddElasticsearch("Elasticsearch", port: 9200)
+    .WithImageTag("8.17.0")
```

Comment:
We have consts for these, but I'd prefer to just have them here like this.

Reply:
I'm going to let them use default so we only have to update the version in one place for tests and app host.